KPI/L.adoc

```diff
@@ -1,73 +1,151 @@
+which it is describing.
 
-Metadata records should contain some information regarding how to get access to the data.
-This information is located in the distributionInfo Element and describe the data format, the service providing access to the data including the access URL as well as the distributorContact when needed. 
+=== Status
 
-==== E.1 To which WCMP element(s) does it apply?
+Draft
 
-To the TemporalElement element in gmd:distributionInfo
+=== WMCP element(s)
 
-//gmd:MD_Metadata/gmd:distributionInfo
+* `//gmd:distributionInfo//gmd:MD_Format`
+* `//gmd:distributionInfo//gmd:MD_DigitalTransferOptions//gmd:onLine//gmd:URL`
+* `//gmd:distributionInfo//gmd:MD_Distributor//gmd:organisationName`
+* `//gmd:distributionInfo//gmd:MD_Distributor//gmd:contactInfo//gmd:electronicMailAddress/gco:CharacterString`
 
-Example:
-....
+
+```xml
 <gmd:distributionInfo>
-    <gmd:MD_Distribution>
-        <gmd:distributionFormat>
-            <gmd:MD_Format>
-<gmd:name> <gco:CharacterString>GRIB</gco:CharacterString>
-                </gmd:name>
-                <gmd:version>
-<gco:CharacterString>FM 92 GRIB Edition 2</gco:CharacterString> </gmd:version>
-<gmd:specification> <gco:CharacterString>http://www.wmo.int/pages/prog/www/WMOCodes.html</gco:CharacterString>
-                </gmd:specification>
-            </gmd:MD_Format>
-        </gmd:distributionFormat>
-        <gmd:transferOptions>
-<gmd:MD_DigitalTransferOptions> <gmd:onLine>
-                    <gmd:CI_OnlineResource>
-                        <gmd:linkage>
-<gmd:URL>ftp://data-portal.ecmwf.int/</gmd:URL> </gmd:linkage>
-<gmd:protocol> <gco:CharacterString>WWW:DOWNLOAD-1.0-ftp--download</gco:CharacterString>
-                        </gmd:protocol>
-                        <gmd:name>
-<gco:CharacterString>ECMWF DCPC FTP Server</gco:CharacterString> </gmd:name>
-                        <gmd:description>
-DRAFT – Guidance on WIS Discovery Metadata (following CBS-16), p36
-</gco:CharacterString> <gco:CharacterString>WMO Information System download service through ECMWF
-                        </gmd:description>
-                        <gmd:function>
-<gmd:CI_OnLineFunctionCode codeList="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/Codelist/gmxCodel
-ists.xml#CI_OnLineFunctionCode" codeListValue="download"> </gmd:function>
-                    </gmd:CI_OnlineResource>
-                </gmd:onLine>
-</gmd:MD_DigitalTransferOptions> </gmd:transferOptions>
-    </gmd:MD_Distribution>
+  <gmd:MD_Distribution>
+    <gmd:distributionFormat>
+      <gmd:MD_Format>
+        <gmd:name>
+          <gco:CharacterString>FM 94 (BUFR)</gco:CharacterString>
+        </gmd:name>
+        <gmd:version>
+          <gco:CharacterString>XII EXT.</gco:CharacterString>
+        </gmd:version>
+        <gmd:specification>
+          <gco:CharacterString>http://www.wmo.int/pages/prog/www/WMOCodes.html</gco:CharacterString>
+        </gmd:specification>
```

Comment:
Comment from my GISC Offenbach colleagues:
- perhaps using xlink inside distributionFormat instead of specification/CharacterString for the url
http://www.wmo.int/schemas/wmdr/1.0RC4/documentation/schemadoc/schemas/distribution_xsd/elements/distributionFormat.html

Reply:
Great suggestion @jsieland 👍.  Added to KPI.